install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# This is an install script for notes
+
+function assertInstalled() {
+    for var in "$@"; do
+        if ! which $var &> /dev/null; then
+            echo "$var is required but not installed, exiting."
+            exit 1
+        fi
+    done
+}
+
+# Yay, Echo self documents! :D
+echo "Checking for root..."
+    [ "$(whoami)" != "root" ] && exec sudo -- "$0" "$@"
```

Comment:
`sudo` doesn't (typically) preserve environmental variables, which means with this script you currently can't uninstall unless you're already running as root. If you're not, then `export uninstall=1; ./install.sh` will elevate itself, and then run a normal install.